README.md

```diff
@@ -49,7 +49,29 @@ Audio files for [Yaapu Telemetry Script and Widget](https://github.com/yaapu/Frs
 
 ## Voices
 
-All of the voices used in the EdgeTX voice packs have been picked from the [neural voices](https://docs.microsoft.com/en-us/azure/cognitive-services/speech-service/language-support?tabs=speechtotext#prebuilt-neural-voices) offered by Microsft Azure text to speech service, in order to get as close as possible to humanlike voices. If you want to see what voices are available, and try different phrases, [check out the online demo generator](https://azure.microsoft.com/en-us/services/cognitive-services/text-to-speech/#features). Using some recording software, you could even save your own phrases and use them in the voice packs.
+All of the voices used in the EdgeTX voice packs have been picked from the [neural voices](https://docs.microsoft.com/en-us/azure/cognitive-services/speech-service/language-support?tabs=speechtotext#prebuilt-neural-voices) offered by Microsoft Azure text to speech service, in order to get as close as possible to human-like voices. If you want to see what voices are available, and try different phrases, [check out the online demo generator](https://azure.microsoft.com/en-us/services/cognitive-services/text-to-speech/#features). Using some recording software, you could even save your own phrases and use them in the voice packs.
+
+### Generating custom phrases
+If you have a [Azure Speech Services subscription](https://azure.microsoft.com/pricing/details/cognitive-services/speech-services/) (there is a free usage tier), phrases can be generated with `curl` or a `http` client like `postman`. After building a text to speech resource in `Azure` you can use it by `REST` calls (`http` requests).
+
+The request url is:
+`https://<YOUR_RESOURCE_REGION>.tts.speech.microsoft.com/cognitiveservices/v1`
+
+You should add the following headers to your request (EdgeTX supports up to 32khz `.wav` file but in that range 8khz is the highest value supported by the conversion service. However, it is possible to select higher quality like `riff-48khz-16bit-mono-pcm` and convert to 32khz afterwards with another tool):
+
+```
+Ocp-Apim-Subscription-Key: <YOUR_RESOURCE_KEY>
+Content-Type: application/ssml+xml
+X-Microsoft-OutputFormat: riff-8khz-16bit-mono-pcm
+```
+
+And in the request body (raw) place your `ssml` (change the voice name according to your preference, the full list is: `tts.speech.microsoft.com/cognitiveservices/voices/list`):
```

Comment:
@shkup And the full list is accessed how? It's not quite the same as say a link like
https://learn.microsoft.com/en-us/azure/cognitive-services/speech-service/language-support?tabs=stt-tts

Reply:
Possible audio outputs are listed here:
 `https://learn.microsoft.com/en-us/azure/cognitive-services/speech-service/rest-text-to-speech?tabs=nonstreaming#audio-outputs`
 But I think that the best approach is always passing `riff-48khz-16bit-mono-pcm` and later on converting it with `ffmpeg` using the command `ffmpeg -i input.wav -ar 32000 output.wav` as it yields the highest possible quality in EdgeTX (16 bits, 32Khz).
 I tested and it worked perfectly.

Reply:
I'd have to double-check, should be able to incorporate as well ... any improvement is good, right? :grin: Although I do wonder if it would really be noticeable on these radios - maybe only with headphones?

Reply:
Ok, tweaked the documentation a bit more, so lets's see how that goes. Don't hesitate to point anything you think needs changing... I'll only grumble and probably do it anyway :grin: